src/main/java/de/tum/in/www1/artemis/domain/quiz/ShortAnswerSubmittedText.java

```diff
@@ -80,7 +80,13 @@ public void setSubmittedAnswer(ShortAnswerSubmittedAnswer shortAnswerSubmittedAn
      * @return boolean true if submittedText fits the restrictions above, false when not
      */
     public boolean isSubmittedTextCorrect(String submittedText, String solution) {
-        return FuzzySearch.ratio(submittedText.toLowerCase().trim(), solution.toLowerCase().trim()) > 85;
+        ShortAnswerQuestion saQuestion = submittedAnswer != null ? (ShortAnswerQuestion) submittedAnswer.getQuizQuestion() : new ShortAnswerQuestion();
+        boolean matchLetterCase = saQuestion.matchLetterCase() != null && saQuestion.matchLetterCase();
```

Comment:
I think you can replace this with the following e.g.:

`Objects.requireNonNullElse(saQuestion.matchLetterCase(), false)`